src/main/resources/openapi/tasks.yaml

```diff
@@ -132,6 +132,8 @@ components:
           $ref: '#/components/schemas/CronExpression'
         is_active:
           $ref: '#/components/schemas/IsActive'
+        batchSize:
+          $ref: '#/components/schemas/batchSize'
```

Comment:
Please capitalise this to BatchSize for consistency with the other component names.

Reply:
done